src/main.rs

```diff
@@ -324,14 +336,20 @@ fn test_merge_avgs() {
     let mut avgs: BTreeMap<String, Vec<_>> = BTreeMap::new();
 
     let avg1 = avg::AVG {
-        issues: vec!["CVE-1".to_string(), "CVE-2".to_string()],
+        issues: vec![
+            ("CVE-1".to_string(), enums::Severity::Unknown),
+            ("CVE-2".to_string(), enums::Severity::Unknown),
+        ],
         fixed: Some("1.0.0".to_string()),
         severity: enums::Severity::Unknown,
         status: enums::Status::Fixed,
     };
 
     let avg2 = avg::AVG {
-        issues: vec!["CVE-4".to_string(), "CVE-10".to_string()],
```

Comment:
Why is this change needed?

Reply:
I changed issue in `src/avg.rs`
```rust
-    pub issues: Vec<String>,
+    pub issues: Vec<(String, enums::Severity)>,
```
to print the severity color.

Reply:
Sorry, I meant the different CVE name.